Gemfile

```diff
@@ -221,3 +220,5 @@ gem "maintenance_tasks", "~> 2.3"
 gem "stackprof"
 gem "sentry-ruby"
 gem "sentry-rails"
+# for SAML based SSO
+gem 'devise_saml_authenticatable', '~> 1.9.1'
```

Comment:
I do not think there is a need for two entries for `gem 'devise_saml_authenticatable'`, lets remove the entry without version here. Run bundle install again and also commit updated `Gemfile.lock`